developer_manual/general/devenv.rst

```diff
@@ -55,9 +55,10 @@ After the development tool installation make the directory writable::
 Then install Nextcloud from git::
 
   git clone git@github.com:nextcloud/server.git --branch $BRANCH /var/www/<folder>
```

Comment:
`-branch $BRANCH` is not needed. Master is used by default and then git can do the rest.